docs/starting/configuring.rst

```diff
@@ -9,7 +9,7 @@ behavior.
 
 ``WAFFLE_COOKIE``
     The format for the cookies Waffle sets. Must contain ``%s``.
-    Defaults to ``dwf_%s``.
+    Defaults to ``dwft_%s``.
```

Comment:
This is actually a different cookie than `dwft_`, this is the non-"testing" version.